src/content/getting-started/cloud-provider-accounts/cluster-api/azure/index.md

```diff
@@ -0,0 +1,141 @@
+                     --location <AzureRegion> \
+                     --template-file delegatedResourceManagement.json \
+                     --parameters delegatedResourceManagement.parameters.json \
+                     --verbose
+```
+
+You will have to supply a general Delegated Resource Management [template file](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.json).
+
+The Delegated Resource Management template uses a [parameters file](https://raw.githubusercontent.com/giantswarm/azure-operator/master/docs/delegatedResourceManagement.parameters.json) to supply the needed variables for configuration.
+
+__Warning__: Please remember to change the `roleDefinitionId` in case you would like to use your custom role definition. Moreover, ask your Account Engineer so he can provide you with the `GiantSwarmPrincipalID` and `GiantSwarmTenantID` values.
+
+### Accept legal terms for the deployment of Flatcar image
+
+Giant Swarm deploys [Flatcar](https://www.flatcar-linux.org/) image developed by Kinvolk from Azure Marketplace. In order to be able to run the image, it is required by Azure to accept the legal terms.
+Please run the following command prior to creating a cluster on a given subscription:
+
+```nohighlight
+az vm image terms accept --offer flatcar-container-linux-free --plan stable --publisher kinvolk
+```
+
+This acceptance should be performed once for a subscription that is used to run Giant Swarm workload clusters.
+
+### Configure Azure subscription configuration for Management and Workload Cluster deployments
+
+Currently implementation of management cluster's bootstrap process with Cluster API for Azure (CAPZ) requires a [Azure Service Principal](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals?tabs=browser#service-principal-object) which we will further refer to as _Giant Swarm Service Principal_. This service principal is required to be configured with all necessary permissions for creation and managing Azure resources.
+
+During bootstrapping process a [User Assigned managed identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp) is created that is from that point used for any interactions with the cloud resources.  
+
+This identity can:
+
+* Create Workload Clusters, and all the related resources, in the `same subscription` where the MC lives
+* Be `authorized by the Customer` to create Workload Clusters, and all the related resources, in _other subscriptions_ belonging to the same tenant - __NOTE__ _this is not yet implemented_
+* Be `authorized by the Customer` to create Workload Clusters, and all the related resources, in _subscriptions that exist in other tenants_ - __NOTE__ _this is not yet implemented_
+
+### Create management cluster service principal for bootstrapping
```

Comment:
This is out of place. We already explain it below, and also there shouldn't be any hints about unimplemented features.

```suggestion
### Create management cluster service principal for bootstrapping
```